__main__.py

```diff
@@ -97,7 +108,7 @@ def on_event(event):
                         conversation))
 
     def show_event(event):
-        return lambda: print(event.text)
+        return lambda: logger.info('Received message: %s', event.text)
```

Comment:
I was avoiding making _too_ many changes, but fine :stuck_out_tongue: I'm now only logging the message when we actually process it. I also removed the call to `loop.call_soon()` when logging the message because it's not actually needed (logging isn't super cheap, but we're also not expecting high volume).